src/Logger.hs

```diff
@@ -20,7 +20,5 @@ data LogEntry =
   | LogEntryForModerator EventLogItemCold
   deriving (Eq, Show)
 
-type SendLogMsg = LogEntry -> IO ()
```

Comment:
1. the type is as long as the synonym.
1. it contains the structure that tells what it means, instead of a name that we cannot trust and that is more ambiguous to begin with.
1. if we have the synonym, we risk not using it and mixing the type with the synonym in different places in the code, which makes reading and refactoring harder.

what are your reasons for keeping it?